pkgs/top-level/python-packages.nix

```diff
@@ -11319,6 +11319,38 @@ in {
     };
   });
 
+  notify2 = buildPythonPackage rec  {
+    pname = "notify2";
+    name = "${pname}-${version}";
+    version = "0.3.1";
+
+    doCheck = false;        # Requires real X session to run tests
+
+    src = pkgs.fetchurl {
+      url = "mirror://pypi/${builtins.substring 0 1 pname}/${pname}/${name}.tar.gz";
+      sha256 = "0z8rrv9rsg1r2qgh2dxj3dfj5xnki98kgi3w839kqby4a26i1yik";
+    };
+
+#    propagatedBuildInputs = [
+#      self.neovim
+#      self.click
+#      self.pygobject3
+#      pkgs.gobjectIntrospection
+#      pkgs.makeWrapper
+#      pkgs.gtk3
+#    ];
+#
+#
+    propagatedBuildInputs = [ self.pygobject3 self.dbus-python pkgs.gtk3 pkgs.gobjectIntrospection ];
```

Comment:
Are you sure about `gtk3` in `propagedBuildInputs`? It looks like it only uses Python libraries and gtk3 does not contain any.

Reply:
I couldn't get it to build without it but I am not positive it is needed. I will do more testing now.